apps/protocol-frontend/src/contexts/UserContext.tsx

```diff
@@ -87,6 +95,19 @@ export const UserContextProvider: React.FC<UserContextProps> = ({
     }
   }, [address]);
 
+  const getUserDaos = useCallback(() => {
+    if (userData) {
+      const userDaos = userData?.guild_users;
+      setUserDaos(userDaos);
+    }
+  }, [userData]);
+
+  const isUserDaoMember = (daoId: string): boolean | undefined => {
```

Comment:
This'll be removed if we prefer the hook

Reply:
Ah! Sounds good. I originally had it as a value from mapping over the values coming from user data but didn't use a Map. I'll make that change and keep all of this in context and remove the extra hook. Thanks for the feedback!